app/docker/components/datatables/containers-datatable/containersDatatableController.js

```diff
@@ -13,7 +13,8 @@ function (PaginationService, DatatableService, EndpointProvider) {
     noStoppedItemsSelected: true,
     noRunningItemsSelected: true,
     noPausedItemsSelected: true,
-    publicURL: EndpointProvider.endpointPublicURL()
+    publicURL: EndpointProvider.endpointPublicURL(),
+    endpointStatus: EndpointProvider.endpointStatus()
```

Comment:
Related to my comment above, instead of tying this component (and the others that requires this information) to the `EndpointProvider`, I believe that we should relay on a binding (such as `offlineMode` as specified above) to inject this property in the component at a higher level.

This will probably makes things easier for us if we decide that we don't way to relay on `EndpointProvider` anymore at some point in time.

Should be applied for all components actually using this logic.